libs/model-assessment/src/lib/ModelAssessmentDashboard/Cohort/ProcessPreBuiltCohort.ts

```diff
@@ -0,0 +1,231 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  ErrorCohort,
+  JointDataset,
+  IFilter,
+  ModelTypes,
+  FilterMethods,
+  Cohort,
+  IPreBuiltFilter
+} from "@responsible-ai/core-ui";
+
+import { IModelAssessmentDashboardProps } from "../ModelAssessmentDashboardProps";
+
+export enum CohortColumnNames {
+  PredictedY = "Predicted Y",
+  TrueY = "True Y",
+  Index = "Index",
+  ClassificationOutcome = "Classification outcome",
+  RegressionError = "Error"
+}
+
+export function processPreBuiltCohort(
+  props: IModelAssessmentDashboardProps,
+  jointDataset: JointDataset
+): [ErrorCohort[], string[]] {
+  const errorStrings: string[] = [];
+  const errorCohortList: ErrorCohort[] = [];
+  if (props.cohortData !== undefined) {
```

Comment:
in typescript xxx !== undefined can be rewrite to  if(!xxx) which will capture other empty values eg null.

However for string and number it may tricky, because !"" == true and !0 == true

Reply:
Changed it to if(props.cohortData)